src/pages/VenuePage/VenuePage.tsx

```diff
@@ -252,6 +258,55 @@ const VenuePage = () => {
     }
   }, [user, profile, venueId, venueTemplate, mixpanel]);
 
+  // Need a token to access the venue
+  useEffect(() => {
```

Comment:
This feels like it should be abstracted somewhere other than here, at the very least encapsulated in a custom react hook.

Reply:
This was refactored into `src/hooks/useVenueAccessToken.ts` in https://github.com/sparkletown/sparkle/pull/1016/commits/3a4a3d7ec1e2f6519931827554f4c7b79bf735cf